add OGPsetting & fix bloglistMV

iput2020-kodama opened the pull request:
OGP設定、ブログリストMVの修正

MORIMOTO520212 committed:
Update ブログ一覧をWordPress化

MORIMOTO520212 committed:
Update SNSアカウント設定フォームの修正

iput2020-kodama committed:
add OGPsetting & fix bloglistMV

OGP設定、ブログリストMVの修正

fumaSuzuki commented:
ブログ一覧ページのスマホ画面で画像のサイズがと見えている部分がデザインと違うから、その部分を直して欲しい！
あとcontent-single.htmlにもその変更内容を反映させておいて欲しいです。

MORIMOTO520212 committed:
Merge branch 'winc1980:main' into main

MORIMOTO520212 committed:
Merge branch 'main' into pr/39

MORIMOTO520212 commented:
@iput2020-kodama 
コンフリクトは解消しました。
指摘事項を修正してコミットしてください。

iput2020-kodama committed:
fix bloglist.html content-single.html

ブロブリストのMVの表示を修正
bloglist.htmlとcontent-single.htmlとも同内容
bloglist.htmlの画像がスマホとPCで違っていた為追加した
